releasenotes-builder/src/test/java/com/github/checkstyle/templates/TemplateProcessorTest.java

```diff
@@ -229,6 +229,15 @@ public void testGenerateOnlyGitHub() throws Exception {
         assertFile("githubPageBreakingCompatibility.txt", MainProcess.GITHUB_FILENAME);
     }
 
+    @Test
+    public void testNoEmptyLineInGitHub() throws Exception {
+        final String lines = getFileContents(new File(MainProcess.GITHUB_FILENAME));
```

Comment:
you need to run `MainProcess.runPostGenerationAndPublication`
and validate something like
`assertFile("githubPageBreakingCompatibility.txt", MainProcess.GITHUB_FILENAME);`

to let me see generated content in gituhb as file.

Reply:
but it says that no such file exists. So should I use the `full path` or `MainProcess.GITHUB_FILENAME`

Reply:
Use full path, I just gave example of how file content is validated

Reply:
Used, please check  the pushed code